public/class-gdpr-cookie-tracking.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * GDPR Cookie Tracking
+ *
+ * @package Gdpr
+ */
+
+declare( strict_types=1 );
+
+/**
+ * Class Gdpr_Cookie_Tracking
+ *
+ * @package Gdpr
+ */
+class Gdpr_Cookie_Tracking {
+
+	public const OPTIONAL_AUTOMATTIC_COOKIE = 'tk_ai';
+	public const OPTIONAL_ANALYTICS_COOKIE = '_ga';
+
+	/**
+	 * Removes code that calls third party tracking scripts
+	 *
+	 * @return void
+	 */
+	public function remove_tracking_scripts() : void {
+		add_filter( 'jetpack_honor_dnt_header_for_stats', '__return_true' );
+
+		/**
+		 * First party
+		 * Disable Google Tag Manager
+		 * Enabled in wp-content/themes/peake/inc/google-tag-manager.php
+		 */
+		remove_action( 'wp_head', 'Peake\Theme\Inc\render_tagmanager_head' );
```

Comment:
Peake namespace shouldn't be referenced in the plugin?

Reply:
Yeah, you're absolutely right. Replaced with a `do_action`, which will be hooked into from Peake.

Corresponding Peake PR - https://github.com/boxuk/peake-wp-content/pull/765

Reply:
Lovely!